documentation/src/pages/sveltekit/start-here/quick-start.md

```diff
@@ -223,20 +210,20 @@ You can get the current user using `getUser()`. Notice that the `username` prope
 ### Redirect unauthenticated user
 
 ```ts
-// +page.ts
-import { redirect } from "@sveltejs/kit";
-import { getUser } from "@lucia-auth/sveltekit/load";
-import type { PageLoad } from "./$types";
-
-export const load: PageLoad = async (event) => {
-	const user = await getUser(event);
-	if (!user) throw redirect(302, "/login");
+// routes/+page.server.ts
+import { redirect } from '@sveltejs/kit';
+import type { PageServerLoad } from './$types';
+
+export const load: PageServerLoad = async ({ locals }) => {
+	const { user } = await locals.getSessionUser();
+	if (!user) throw redirect(302, '/login');
 };
+
 ```
 
 ### Sign out
 
-Add a button that calls [`signOut()`](/sveltekit/api-reference/client-api#signout). Call [`invalidateAll()`](https://kit.svelte.dev/docs/modules#$app-navigation-invalidateall) afterward to re-run load functions and update the session in the client so that the user gets redirected to the login page. `invalidateAll()` for updating the session after sign out will only work if it was preceded by `signOut()`.
+Add a button that calls [`signOut()`](/sveltekit/api-reference/client-api#signout) and [`invalidateAll()`](https://kit.svelte.dev/docs/modules#$app-navigation-invalidateall). `invalidateAll()` will cause all of the load functions to re-run, which will update the session in the client so that the user gets redirected to the login page. `invalidateAll()` must only be called _after_ `signOut()` so that the session may be updated properly.
```

Comment:
You can remove the last sentence (invalidateAll must only be called…)

Reply:
Definitely agree - I kept it incase you were emphasizing for a particular reason